.travis.yml

```diff
@@ -97,6 +97,12 @@ jobs:
 
     # Run Spark integration test and unit test
 
+    # Run spark integration of in one zeppelin instance (3.0)
+    - jdk: "openjdk8"
+      dist: xenial
+      env: BUILD_PLUGINS="true" PYTHON="3" SCALA_VER="2.12" PROFILE="-Phadoop2 -Pintegration" R="true" BUILD_FLAG="install -DskipTests -DskipRat" TEST_FLAG="test -DskipRat" MODULES="-pl zeppelin-interpreter-integration,zeppelin-web,spark/spark-dependencies,markdown -am" TEST_PROJECTS="-Dtest=ZeppelinSparkClusterTest30,SparkIntegrationTest30 -DfailIfNoTests=false"
```

Comment:
Maybe we should also add a test with hadoop3.

Reply:
Thanks for the suggestion, that's make sense, I have added test with hadoop3 in `SparkIntegrationTest30` and `ZeppelinSparkClusterTest30`